### ISSUE TITLE
Reset Download-Counter

### ISSUE THREAD

**Issue by Hoerli1337** [shmykelsa/AAAD#191]
Hi!
I would like to test the app once and if it works, I can spend a few euros for it.
However, the download limit is set totally crappy.
Screen2Auto -> Download does not work
AA Mirror -> Download runs but then error when parsing the application.
=> Download limit reached :(

How can I reset the limit to at least test an app successfully?
I don't feel like waiting 1 month for it now.
Furthermore, I would like to know what my device is authenticated with to determine the download limit.

- Samsung Galaxy Note 8
- Android 12
- LineageOS 19
- microG apps (Aurora Store, no Play Store)

**Comment by shmykelsa** [comment]
Please contact help team through the app and we'll help you sort it out:

- Open AAAD
- Select top right menu
- Select "Help"
- Hit yes

Thank you

**Comment by omaralwardian** [comment]
Hello, I emailed you three times now.

I need to know how to reinstall an app I have currently installed on phone, when it says no download remaining.

**Comment by Hoerli1337** [comment]
Cool. The address is empty, which is passed to the mail program.
So no contact possible.

**Comment by omaralwardian** [comment]
How do we update S2A or redownload when 0 downloads are remaining. 
![Screenshot_20220414-184454_Email](https://user-images.githubusercontent.com/70217313/163481973-1d72db1c-85f5-4596-92c8-af3d8b2b2217.jpg)

**Comment by shmykelsa** [comment]
> omaralwardian

AAAD on free plan allows a download every 30 days. So if you made a download in the last 30 days, you won't be able to download again unless switching to pro.

**Comment by shmykelsa** [comment]
> Cool. The address is empty, which is passed to the mail-app. So no contact possible.

That's not true. Probably you have a bad e-mail client that is not parsing the URL correctly.

**Comment by nnhthuan** [comment]
Most of people come here to test if they can stream video in android auto, I guess. I have 2 phones, I downloaded AAAD and tested both apps (Carstream and Fermato auto), both of them don't work with latest android auto.
I would not recommend you guys to downgrade android auto only for this reason.

**Comment by ucibaris** [comment]
Hi, I have downloaded this tried to install  the first link uoutube downloaded but didn't  work uninstalled and try to download it back have again no available download  option like above comment. Can you help?
